### PR TITLE
CORE-20571 - Change OpenAPI representation of `JsonObject`

### PR DESCRIPTION
**Pull request by filipesoliveira**

### CODE REVIEW

File: libs/rest/rest-server-impl/src/main/kotlin/net/corda/rest/server/impl/apigen/processing/openapi/schema/SchemaModelToOpenApiSchemaConverter.kt [hunk 1]
```diff
@@ -32,7 +32,7 @@ private val log =
 object SchemaModelToOpenApiSchemaConverter {
     @Suppress("ComplexMethod")
     fun convert(schemaModel: SchemaModel): Schema<Any> {
-        log.trace { """Convert schemaModel "$schemaModel" to schema.""" }
+        log.info ("""Convert schemaModel "$schemaModel" to schema.""" )
```
Comment (filipesoliveira) [inline libs/rest/rest-server-impl/src/main/kotlin/net/corda/rest/server/impl/apigen/processing/openapi/schema/SchemaModelToOpenApiSchemaConverter.kt]: This change must be reverted!

File: libs/rest/rest-server-impl/src/main/kotlin/net/corda/rest/server/impl/apigen/processing/openapi/schema/SchemaModelToOpenApiSchemaConverter.kt [hunk 2]
```diff
@@ -65,7 +65,7 @@ object SchemaModelToOpenApiSchemaConverter {
                 `$ref`(schemaModel.ref)
             }
             is JsonSchemaModel -> ComposedSchema().apply {
-                description = "Can be any value - string, number, boolean, array or object."
+                description = "Can be any value - string, number, boolean, array or object." // THis need to be udpate it can only be an object
```
Comment (filipesoliveira) [inline libs/rest/rest-server-impl/src/main/kotlin/net/corda/rest/server/impl/apigen/processing/openapi/schema/SchemaModelToOpenApiSchemaConverter.kt]: The description must be updated and the comment removed.

**Comment by vkolomeyko** [comment]
Looks like this PR needs a rebase to the latest state of the target branch or else the change set looks unexpectedly large.

File: libs/rest/rest-server-impl/src/main/kotlin/net/corda/rest/server/impl/apigen/processing/openapi/schema/SchemaModelToOpenApiSchemaConverter.kt [hunk 3]
```diff
@@ -65,18 +61,9 @@ object SchemaModelToOpenApiSchemaConverter {
                 `$ref`(schemaModel.ref)
             }
             is JsonSchemaModel -> ComposedSchema().apply {
-                description = "Can be any value - string, number, boolean, array or object."
+                description = "JSON object."
```
Comment (vkolomeyko) [inline libs/rest/rest-server-impl/src/main/kotlin/net/corda/rest/server/impl/apigen/processing/openapi/schema/SchemaModelToOpenApiSchemaConverter.kt]:
```suggestion
                description = "Either nested JSON object or a valid JSON-escaped string."
```

**Comment by filipesoliveira** [comment]
build e2e

**Review by vkolomeyko — APPROVED** [review]
LGTM

**Pull request merged**